resource-types/s3-object/src/awscommunity_s3_object/handlers.py

```diff
@@ -574,6 +587,33 @@ def _get_session_client(
     return None
 
 
+def _get_tags_from_previous_resource_tags(
+    previous_resource_tags: Mapping[str, Any],
+) -> List[Dict[str, Any]]:
+    """Create and return a list of tags from request.previousResourceTags."""
+    LOG.debug("_get_tags_from_previous_resource_tags()")
+
+    tags = [
+        {
+            "Key": previous_resource_tag,
+            "Value": previous_resource_tags[previous_resource_tag],
+        }
+        for previous_resource_tag in previous_resource_tags
+    ]
+    return tags
+
+
+def _get_tags_from_model_tags(
+    model_tags,
+) -> List[str]:
+    """Create and return a list of tags from model.Tags."""
+    LOG.debug("_get_tags_from_model_tags()")
+    tags = []
+    for model_tag in model_tags: 
+        tags += [model_tag.Key + "=" +model_tag.Value]
+    return tags
+
+
 def _get_model_tags_from_tags(
```

Comment:
Also looks like there are no references, but maybe we will need this for our Read handler?